Services/Common/OJS.Workers/OJS.Workers.ExecutionStrategies/Java/JavaProjectTestsExecutionStrategy.cs

```diff
@@ -415,6 +427,11 @@ protected virtual void ExtractUserClassNames(string submissionFilePath)
                 if (!isTestSuccessful)
                 {
                     var errorLine = output.ReadLine();
+                    if (errorLine!.StartsWith(InvalidNumberOfTestCasesPrefix))
```

Comment:
Can this be null? If null, we can handle it and return a better exception, instead of "null reference"

Reply:
Done.